src/js/components/Layout/Menu.jsx

```diff
@@ -1,77 +1,78 @@
-import React from 'react';
+import React, { useMemo } from 'react';
 
 import _ from 'lodash';
 import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
+import { withRouter } from 'react-router-dom';
 
+import { getMenuItemsComponent, getSectionComponent, getSubsectionComponent } from 'utils/MenuUtils';
+
+
+const Menu = ({ menuConfig, location }) => {
+  const sectionsToExclude = ['Analytics', 'Configuration'];
```

Comment:
At first side I didn't want to do it on the backend side, as I wanted to avoid touching menu on gsp pages yet
Is `def getMenuConfig` in `ApiController` common for react and gsp? If it isn't, I could move it to backend.

Reply:
I probably answered myself - I think that for gsp it uses `def megamenu` from `DashboardController`. If so, I'll try to move those `sectionsToExclude` to backend

Reply:
I believe that `config.openboxes.megamenu` is a shared config between React and gsp.
There is also [`enabled` property on megamenu items](https://github.com/openboxes/openboxes/blob/develop/grails-app/conf/Config.groovy#L1403)  where you can probably set it to false and it will filter out this item from the menu

Reply:
yeah, you're right, but if we don't want to touch gsp yet, we shouldn't make `enabled = false`, so I was just thinking about filtering it inside `def getMenuConfig`, so the megamenu would not be touched yet. 
Anyway we are going to remove it on gsp soon too, so maybe we could just proceed with this `enabled = false` even if those sections hide on gsp just now? 
@awalkowiak thoughts on that?